registration_form/registration_form.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <title> Форми </title>
+    <script src="registration_form.js"></script>
+</head>
+
+<body>
+    <form name="form" method="post" action="register.php" onsubmit="return validate();">
+        <fieldset>
+            <legend> Информация за избираемата дисциплина </legend>
+            <label for="username">Потребителско име:</label>
+            <input id="username" name="username" require>
```

Comment:
You may want to add validations on the html inputs. Giving feedback to the user as early as possible is always better and this will save him time. You could check here for a reference: https://github.com/martinely95/WEB2020-php/blob/master/ex5/index.html